definitions/velocity.proto

```diff
@@ -42,3 +42,57 @@ message DetectMotionRequest {
   required PlatformType platform  = 6;
   required double timestamp       = 7;
 }
+
+message HandshakeRequest {
+  required string authToken       = 1;
+  optional string ifa             = 2;
+  optional string userId          = 3;
+  optional string appId           = 4;
+  optional PlatformType platform  = 5;
+}
+
+
+message HandshakeResponse {
+  optional double sampleSize      = 1;
+  optional double captureInterval = 2;
+  optional bool   canDetectMotion = 3;
+  optional bool   canLabelMotion  = 4;
+}
+
+
+message SensorsRequest {
+  required string impressionId    = 1;
+  repeated Sensor sensors         = 2;
+  optional uint32 sequenceIndex   = 3;
+  repeated string modelNames      = 4;
+  optional string ifa             = 5;
+  optional string userId          = 6;
+  optional string appId           = 7;
+  optional string metadata        = 8;
+  optional double timestamp       = 9;
+  optional PlatformType platform  = 10;
```

Comment:
We don't need any of the optional fields. Most of them we get from the handshake and sequence index in case of WebSockets can be counted on server. Even impression id is no longer necessary. I'd like to start with minimal and add stuff as we find necessary since adding more optional fields is backwards compatible anyway.

Reply:
I would like to keep `impressionId`, I think, that it would be useful on reconnect case.